cli/command/cli.go

```diff
@@ -265,11 +239,6 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions, ops ...Initialize
 		}
 	}
 	cli.initializeFromClient()
-
-	if err := cli.loadClientInfo(); err != nil {
```

Comment:
This one was added back in https://github.com/docker/cli/pull/2544, related to https://github.com/docker/cli/issues/2543, which should also no longer be an issue.

That said; we need to verify if there's any codepath that expects the client to perform version negotiation somewhere.

Reply:
Actually looks like that's already handled by `initializeFromClient()`; https://github.com/docker/cli/blob/a32cd16160f1b41c1c4ae7bee4dac929d1484e59/cli/command/cli.go#L361-L370